projects/js-packages/publicize-components/src/social-store/resolvers.js

```diff
@@ -70,11 +69,11 @@ export function* getAutoConversionSettings() {
  * @returns {Function} Resolver
  */
 export function getConnections() {
-	return function ( { dispatch } ) {
+	return function ( { dispatch, registry } ) {
```

Comment:
This clearly works but I'd like to understand more about it. I've not seen resolvers used like this before, and thought they should return action objects as a result of a generator or a promise, not a function.

Reply:
Ah it's a thunk! I assume the registry is needed to make testing easier?

Reply:
Yes, it's a thunk. Yes, registry makes testing easier.